packages/meta-harness/tests/test_evaluator_worktree_lifecycle.py

```
@@ -237,9 +286,178 @@ def fake_materialize(_worktree_dir, _package_dir):
+        assert hard_failure is False, errors
+        assert call_order == [
+            "snapshot_trusted_fixtures",
+            "materialize_fixtures",
+            "run_headless_scenario",
+            "materialize_fixtures",
+        ]
+
+    def test_tamper_restore_materialize_still_runs_after_headless_scenario(
+        self, git_project: Path, monkeypatch
+    ) -> None:
+        """改ざん対策（PR #326 round 4/5）の維持: snapshot 前 materialize を追加しても、
```

Ruff (0.16.0)

[warning] Docstring contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF002)